cvat/apps/engine/media_extractors.py

```diff
@@ -101,6 +101,21 @@ def image_size_within_orientation(img: Image.Image):
 def has_exif_rotation(img: Image.Image):
     return img.getexif().get(ORIENTATION_EXIF_TAG, ORIENTATION.NORMAL_HORIZONTAL) != ORIENTATION.NORMAL_HORIZONTAL
 
+
+def load_image(image: tuple[str, str, str])-> tuple[Image.Image, str, str]:
+    with Image.open(image[0]) as pil_img:
+        pil_img.load()
+
+        # current version of Pillow applies exif rotation immediately when TIFF image opened
+        # and it removes rotation tag after that
+        # so, has_exif_rotation(image) will return False for TIFF images even if they were actually rotated
+        # and original files will be added to the archive (without applied rotation)
+        # that is why we need the second part of the condition
```

Comment:
I still don't understand. If loading the TIFF image removes the Exif orientation tag, then wouldn't `ImageOps.exif_transpose` do nothing anyway? It uses the same tag to determine whether it needs to do anything:

https://github.com/python-pillow/Pillow/blob/10.4.0/src/PIL/ImageOps.py#L687

For that matter, it doesn't seem like we need the first part of the condition either, since `exif_transpose` effectively contains the same condition.